scripts/wasmd/template/.wasmd/config/config.toml

```diff
@@ -259,13 +259,19 @@ max_txs_bytes = 1073741824
 # Size of the cache (used to filter transactions we saw earlier) in transactions
 cache_size = 10000
 
+# Do not remove invalid transactions from the cache (default: false)
+# Set to true if it's not possible for any invalid transaction to become valid
+# again in the future.
+keep-invalid-txs-in-cache = false
```

Comment:
How come this isn't snake_case?

Reply:
Autogenerated by wasmd / cosmos-sdk (like all the config changes)